src/common/settings.js

```diff
@@ -26,6 +26,8 @@ var loadDefault = function(version) {
         showTrayIcon: false,
         trayIconTheme: '',
         disablewebsecurity: true,
+        minimizeToTray: false,
```

Comment:
From reading https://github.com/mattermost/desktop/issues/178 I think this should be ```true``` and therefor the default behavior